stackstorm.yml

```diff
@@ -3,12 +3,17 @@
   hosts: all
   environment: "{{ st2_proxy_env | default({}) }}"
   roles:
-    - StackStorm.mongodb
+    - role: StackStorm.mongodb
+      mongodb_version: 4.0
```

Comment:
:+1: 
Similar to `StackStorm.st2mistral` import, need to add condition here so use `4.0` only for Ubuntu Bionic.

Reply:
will do.